hookman/hooks.py

```diff
@@ -129,3 +137,36 @@ def _bind_libs_functions_on_hook_caller(self, shared_lib_path: Path, hook_caller
         for hook in hooks_to_bind:
             cpp_func = getattr(hook_caller, hook)
             cpp_func(hooks_to_bind[hook])
+
+
+    def plugins_has_conflicts(self) -> bool:
+        """
+        Auxiliary methods that checks if the get_status has any conflict
+        """
+        if self.get_status():
```

Comment:
Shorter: `return bool(self.get_status())`

Reply:
Good call, I didn't know that this was possible =)